_cdn/cdn.yml

```diff
@@ -1409,13 +1409,13 @@ images_products_bpmsuite_bpmsuite.png:
 images_products_bpmsuite_rewards-workshop.png:
   build_no: 1
   md5sum: f8cc5660eaffa471bb3a59c0f01a7f3d
-images_champions_roadtoawesome_graphic.png:
```

Comment:
No need to make any changes to this file. The change in filename will cause cache invalidation.